object/service.proto

```diff
@@ -304,6 +304,8 @@ message SearchRequest {
     //
     // * $Object:version \
     //   version
+    // * $Object:objectID                       \
+    //   object_id
```

Comment:
Are there any cases when it is useful to make search request with `$Object:objectID` filter when you can use less exhaustive `object.Head` request?

Reply:
Search for objects in a group except the known set. For example search for other object parts.

Reply:
`$Object:objectID StringNotEqual <some-object-id>` makes sense. We don't have `StringNotEqual` matcher yet, but if it is planned, then I don't mind having `$Object:objectID ` already in the list.